fix: wrong message `Unauthorized to connect to Openproject` shown with valid SSO connection (#1018)
nextcloud/integration_openproject: CHANGELOG.md

### Fixed

> - Fix: Shown wrong message "Unauthorized to connect to Openproject" on Single-Sign-On setup [#1018](https://github.com/nextcloud/integration_openproject/pull/1018)

Reviewer: :+1:

> - Fix: Shown wrong message "Unauthorized to connect to OpenProject" on Single-Sign-On setup [#1018](https://github.com/nextcloud/integration_openproject/pull/1018)

Reviewer: ```suggestion
- Fix: Wrong message "Unauthorized to connect to OpenProject" shown with valid SSO connection [#1018](https://github.com/nextcloud/integration_openproject/pull/1018)
```
